components/infobox/commons/infobox_person.lua

```diff
@@ -16,21 +24,21 @@ local function GetBirthAndDeath()
 	return '', nil, nil, nil
 end
 
-local Player = Class.new(BasicInfobox)
+local Person = Class.new(BasicInfobox)
 
 local Language = mw.language.new('en')
 local _LINK_VARIANT = 'player'
-local shouldStoreData
+local _SHOULD_STORE_DATA
```

Comment:
Oh I'm sorry, I thought this was a constant. We should set this to self => `self.shouldStoreData`

Reply:
as it is only used within the function where it is set, i think making it a local of the function would be better, right?

Reply:
Sure